extract_assets.py

```diff
@@ -21,7 +22,13 @@ def ExtractFile(xmlPath, outputPath, outputSourcePath):
         # Don't extract if another file wasn't extracted properly.
         return
 
-    execStr = f"tools/ZAPD/ZAPD.out e -eh -i {xmlPath} -b baserom/ -o {outputPath} -osf {outputSourcePath} -gsf 1 -rconf tools/ZAPDConfigs/MqDbg/Config.xml {ZAPDArgs}"
+    zapdPath = Path("tools") / "ZAPD" / "ZAPD.out"
+    configPath = Path("tools") / "ZAPDConfigs" / "MqDbg" / "Config.xml"
+
+    Path(outputPath).mkdir(parents=True, exist_ok=True)
+    Path(outputSourcePath).mkdir(parents=True, exist_ok=True)
```

Comment:
This fixes:
![image](https://user-images.githubusercontent.com/5306619/235380660-db438232-9583-4d9e-bc6a-f3f42bf28b5e.png)

apparently when creating e.g. assets/code/something without assets/code existing first

IMO this is on ZAPD for not creating the directories recursively but apparently it's hard to do safely in C++ ?

In normal circumstances the folder creation is in the Makefile iirc